lib/dataTrigger/DataTrigger.js

```diff
@@ -0,0 +1,61 @@
+const DataTriggerExecutionResult = require('./DataTriggerExecutionResult');
+const DataTriggerExecutionError = require('../errors/DataTriggerExecutionError');
+const DataTriggerInvalidDocumentActionError = require('../errors/DataTriggerInvalidDocumentActionError');
```

Comment:
These two erros seems like not consensus errors. I would move them into `dataTriggers/errors`